navigation.yaml

```diff
@@ -1239,6 +1239,8 @@ iot:
       path: /internet-of-things/networking
     - title: Smart city
       path: /internet-of-things/smart-city
+    - title: Smart home
```

Comment:
I thought we weren't adding more things to large navigation's like this one?

Reply:
@davegoddard42 and I spoke about it yesterday, and he is fine with this temporarily - he's going to be looking at a new design for the new link sections on /iot and the subpages to address this.